src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
```

Comment:
[checklist] 2. Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

40 is a basic const price